### PR TITLE
Change the order of cycling messages.

### PR DESCRIPTION
**Pull request by G61**

### CODE REVIEW

File: src/main/java/in/twizmwaz/cardinal/module/modules/match/MatchModule.java [hunk 1]
```diff
@@ -55,9 +55,9 @@ public void onMatchEnd(MatchEndEvent event) {
 
     @EventHandler(priority = EventPriority.LOWEST)
     public void onCycleComplete(CycleCompleteEvent event) {
+    Bukkit.broadcastMessage(ChatColor.DARK_AQUA + "Cycled to " + ChatColor.AQUA + event.getMatch().getMapInfo().getName());
```
Comment (twizmwazin) [inline src/main/java/in/twizmwaz/cardinal/module/modules/match/MatchModule.java]: The indentation is incorrect

**Comment by twizmwazin** [comment]
What exactly is the benefit of this pr?

**Comment by G61** [comment]
Just the "You joined observers" is appearing before Cycled to x and changing the order.

**Comment by twizmwazin** [comment]
Sure, why not

**Pull request merged**